modules/ossm-migrating-multitenant-with-cert-manager.adoc

```diff
@@ -0,0 +1,229 @@
++
+[source,terminal]
+----
+$ oc label ns bookinfo tenant=tenant-a
+----
++
+[NOTE]
+====
+In {SMProduct} 2.6, namespaces were enrolled into the mesh by adding them to the `Service MeshMemberRoll` resource. In {SMProduct} 3, you must label each one of your dataplane namespaces with the `discoverySelectors.matchLabels` label.
+====
+
+== Migrating workloads
+
+Now you can migrate your workloads from the {SMProduct} 2.6 controlplane to the {SMproduct} 3.0 controlplane.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately.
+====
+
+. Find the current `IstioRevision` for your {SMProduct} 3.0 controlplane by running the following command:
++
+[source,terminal]
+----
+$ oc get istios istio-tenant-a
+NAME             REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+istio-tenant-a   1           1       0        istio-tenant-a    Healthy   v1.24.1   30s
+----
++
+[NOTE]
+====
+The naming format of your revisions depends on which upgrade strategy you choose for your `Istio` instance.
+====
+
+. Copy the `ACTIVE REVISION` to use as your `istio.io/rev` label in the next step.
+
+. Update injection labels on the `dataplane` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc label ns bookinfo istio.io/rev=istio-tenant-a maistra.io/ignore-namespace="true" --overwrite=true
+----
++
+This adds the following labels to the namespace:
++
+* `istio.io/rev: istio-tenant-a` label: Ensures that any new pods that get created in that namespace connect to the {SMProduct} 3.0 proxy.
++
+* `maistra.io/ignore-namespace: "true"` label: Disables sidecar injection for {SMProduct} 2.6 proxies in the namespace so the {SMProduct} 2.6 stops injecting proxies in this namespace, and any new proxies are injected by the {SMProduct} 3.0 `controlplane`. Without this label, the {SMProduct} 2.6 injection webhook tries to inject the pod and it will connect to the {SMProduct} 2.6 proxy as well as refuse to start since it will have the {SMProduct} 2.6 Carrier Infrastructure Network (CNI) annotation.
++
+[NOTE]
+====
+Once you apply the `maistra.io/ignore-namespace` label, any new pod that gets created in the namespace will connect to the {SMProduct} 3.0 proxy. Workloads can still communicate with each other regardless of which control plane they are connected to.
+====
+
+. Use the `curl` command for the `productpage` pod in the `bookinfo` application so proxies can still communicate with one another by running the following command:
++
+[source,terminal]
+----
+$ oc exec -it -n bookinfo deployments/productpage-v1 -c istio-proxy -- curl localhost:9080/productpage
+----
+
+. Restart the workloads:
++
+.. You can restart workloads all at once so that the new pod will be injected with the {SMProduct} 3.0 proxy by running the follow command:
++
+[source,terminal]
+----
+$ oc rollout restart deployments -n bookinfo
+----
+
+.. Alternatively, you can restart each workload individually by running the following command for each workload:
```

Comment:
Same comment here that the general advice is applicable to all workloads but the specific command is only applicable for this example
```suggestion
.. Alternatively, you can restart each workload individually. For this example with bookinfo, run the following command for each workload:
```